Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 15
Rules claiming more threads will be scaled down.
Job stats:
job             count
------------  -------
all                 1
bio_insights        1
conf_factors        1
suppl_files         1
total               4

Select jobs to execute...

[Thu Jul  4 19:47:08 2024]
rule conf_factors:
    input: deconv_analysis_fig2_3fig4_2_3_6_9.html
    output: conf_factors_fig4_7.html
    jobid: 6
    reason: Missing output files: conf_factors_fig4_7.html
    resources: tmpdir=/tmp

[Thu Jul  4 19:47:08 2024]
rule bio_insights:
    input: deconv_analysis_fig2_3fig4_2_3_6_9.html
    output: bio_insights_fig4_4_5.html
    jobid: 7
    reason: Missing output files: bio_insights_fig4_4_5.html
    resources: tmpdir=/tmp

[Thu Jul  4 19:47:08 2024]
rule suppl_files:
    input: deconv_analysis_fig2_3fig4_2_3_6_9.html
    output: suppl_files1_4_5.html
    jobid: 1
    reason: Missing output files: suppl_files1_4_5.html
    resources: tmpdir=/tmp

[Thu Jul  4 19:47:15 2024]
Error in rule conf_factors:
    jobid: 6
    input: deconv_analysis_fig2_3fig4_2_3_6_9.html
    output: conf_factors_fig4_7.html

RuleException:
CalledProcessError in file /data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/snakemake_pipeline, line 55:
Command 'set -euo pipefail;  Rscript --vanilla -e 'rmarkdown::render("/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/.snakemake/scripts/tmpevscqdqy.conf_factors_fig4_7.Rmd", output_file="/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/conf_factors_fig4_7.html", quiet=TRUE, knit_root_dir = "/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4", params = list(rmd="/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/.snakemake/scripts/tmpevscqdqy.conf_factors_fig4_7.Rmd"))'' returned non-zero exit status 1.
  File "/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/snakemake_pipeline", line 55, in __rule_conf_factors
  File "/home/si48met/miniconda3/envs/cfdeconv_lc/lib/python3.10/concurrent/futures/thread.py", line 58, in run
[Thu Jul  4 19:47:16 2024]
Error in rule suppl_files:
    jobid: 1
    input: deconv_analysis_fig2_3fig4_2_3_6_9.html
    output: suppl_files1_4_5.html

RuleException:
CalledProcessError in file /data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/snakemake_pipeline, line 63:
Command 'set -euo pipefail;  Rscript --vanilla -e 'rmarkdown::render("/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/.snakemake/scripts/tmp_phqc6ym.suppl_files1_4_5.Rmd", output_file="/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/suppl_files1_4_5.html", quiet=TRUE, knit_root_dir = "/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4", params = list(rmd="/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/.snakemake/scripts/tmp_phqc6ym.suppl_files1_4_5.Rmd"))'' returned non-zero exit status 1.
  File "/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/snakemake_pipeline", line 63, in __rule_suppl_files
  File "/home/si48met/miniconda3/envs/cfdeconv_lc/lib/python3.10/concurrent/futures/thread.py", line 58, in run
[Thu Jul  4 19:47:16 2024]
Error in rule bio_insights:
    jobid: 7
    input: deconv_analysis_fig2_3fig4_2_3_6_9.html
    output: bio_insights_fig4_4_5.html

RuleException:
CalledProcessError in file /data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/snakemake_pipeline, line 47:
Command 'set -euo pipefail;  Rscript --vanilla -e 'rmarkdown::render("/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/.snakemake/scripts/tmp3gdoegdm.bio_insights_fig4_4_5.Rmd", output_file="/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/bio_insights_fig4_4_5.html", quiet=TRUE, knit_root_dir = "/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4", params = list(rmd="/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/.snakemake/scripts/tmp3gdoegdm.bio_insights_fig4_4_5.Rmd"))'' returned non-zero exit status 1.
  File "/data/fass5/projects/aram/current_projects/phd_tests/Chapter 4/snakemake_pipeline", line 47, in __rule_bio_insights
  File "/home/si48met/miniconda3/envs/cfdeconv_lc/lib/python3.10/concurrent/futures/thread.py", line 58, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-07-04T194706.581161.snakemake.log
